package.json

```diff
@@ -31,6 +31,7 @@
     "bazel:lint": "yarn bazel:format --lint=warn",
     "bazel:lint-fix": "yarn bazel:format --lint=fix",
     "copy:schematics": "ts-node ./build/copy-schematics-core.ts",
+    "schematics:check": "git diff --name-only --exit-code ./modules",
```

Comment:
Was the intention to use this script in the CI?
If it was, then I think we forgot to update the config.yml 😅